addon-test-support/ember-qunit/adapter.js

```diff
@@ -28,10 +28,11 @@ function unhandledRejectionAssertion(current, error) {
 let Adapter = Ember.Test.Adapter.extend({
   init() {
     this.doneCallbacks = [];
+    this.QUnit = this.QUnit || QUnit;
```

Comment:
Just a personal preference, but `this.QUnit` looks weird to me. I'd just use regular casing here - `this.qunit`.

Reply:
updated